packages/eslint-plugin-fiori-tools/docs/rules/line-endings.md

```diff
@@ -0,0 +1,11 @@
+# Detect usage of invalid line endings (line-endings)
+
+Different operating systems usually represent a newline with different characters, for details see here.
+
+For example Windows uses CR+LF (carriage return and line-feed characters) whereas Unix only uses the LF character.
+
+Mixed kinds of line-endings in the same repository can cause code reviews or automatic merging to be .
```

Comment:
`to be ?` we are missing a verb here.

Reply:
This file is not needed. Deleted in https://github.com/SAP/open-ux-tools/pull/3788/changes/46e2935ba9af127f4a0e3269820bdb5934870fc5